pybossa/cache/task_browse_helpers.py

```diff
@@ -352,7 +352,8 @@ def validate_user_preferences(user_pref):
 
     # Support old spelling of Türkiye
     if valid_locations:
-        valid_locations.append('Turkey')
+        valid_locations.append(['Turkey', 'Turkiye'])
+        valid_locations.extend()
```

Comment:
Do we need this if we add all valid versions to config?

Reply:
It would be confusing for the worker to see Türkiye, Turkey, and Turkiye in the location preferences drop down selector. See screenshot of gigwork edit account page:
![image](https://github.com/bloomberg/pybossa/assets/17805819/7ce96903-d561-464a-8797-b9dfb4d7f9af)

 So my solution is:
1. 'Türkiye' in the config, so users can only set their preference to the new version (already implemented in previous ticket)
2. Adding the other 2 spellings in the validation method ONLY (above lines), so we support all 3 spellings for task csv upload

Reply:
This seems to be patch work. if config is supporting one type of name, ideal would be to backfill the database too. However, as this solution has been discussed internally and agreed upon with stakeholders, this solution seems fine as a short term.